src/tests/unit/send-get/strided_sendget.f90

```diff
@@ -0,0 +1,67 @@
+! Written by Andre Vehreschild
+program stridedsendgettest
+
+  implicit none
+
+  integer, save, dimension(4,6) :: srcmat[*], dstmat[*]
+  integer, save, dimension(6) :: srcvec[*], dstvec[*]
+  integer :: i
+  logical :: test_passed = .true.
+
+  if (this_image() == 2) then
+    srcvec = [(2 * i, i = 1, 6)]
+    srcmat = reshape([(i * 2, i = 1, 4*6)], [4,6])
+  elseif (this_image() == 3) then
+    dstmat = -1
+    dstvec = -2
+  end if
+
+  sync all
+  if (this_image() == 1) then
+    dstvec(:)[3] = srcvec(:)[2]
+    dstmat(3,:)[3] = srcvec(:)[2]
+  end if
+
+  sync all
+  if (this_image() == 3) then
```

Comment:
Is there a way to abstract out image numbers 1, 2 and 3? i.e. is there a way to ensure that the test would at least pass (if not actually test for correctness) on 1 or 2 images?